cocos/gfx/webgl/webgl-command-allocator.ts

```diff
@@ -23,7 +23,7 @@
  THE SOFTWARE.
  */
 
-import { CachedArray } from '../../memop/cached-array';
+import { CachedArray } from '../../core';
```

Comment:
Will use sub-folder reduce circular dependency?

Reply:
I didn't find circular dependency when compiling the engine.